manifests/puppet.pp

```diff
@@ -1,7 +1,4 @@
-#
-# simp_options::puppet class
-#
-# Sets up Puppet configuration variables
+# @summary Sets up Puppet configuration variables
```

Comment:
Please make this change to all manifests missing `@summary` and regenerate the REFERENCE.md